_blog/2023-03-01-getting-started-with-rust-and-gpt.mdx

```diff
@@ -296,8 +294,8 @@ This will allow our frontend to compile directly to the static folder that our b
 If we run `npm run build`, assuming there were no build errors we should now have a subfolder in our API directory called "static", which will hold all of our static assets that we can refer to in our Rust project. Let's implement our static folder by using the `shuttle_shared_folder` package:
 
 ```rust
-// lib.rs
-#[shuttle_service::main]
+// main.rs
+#[shuttle_runtime::main]
```

Comment:
Only thing missing is the return type of this function

Reply:
Oh man!  Will fix this evening and resubmit. :)

Reply:
Awesome, thanks! :pray: By the way, to fix it you can just create another commit on this branch, and then push that to `origin`, they all get added to this PR